opentreemap/importer/models/base.py

```diff
@@ -70,6 +70,18 @@ def status_summary(self):
         t = "Unknown Error While %s" if self.is_lost else "%s"
         return t % self.status_description()
 
+    def is_past_verifying_stage(self):
+        # This method is used by tasks for concurrency control
+        self.refresh_from_db()
```

Comment:
I would not expect an `is_*` method that returns a boolean to have the side effect of querying the DB and possibly overriding properties I had edited on the import event model. We should consider moving the refresh call to [`tasks.py:103`](https://github.com/OpenTreeMap/otm-core/pull/2675/files#diff-52bab5e6c25271c634db1a91d4ae5792R104), just before we call `_update_ie_status`